### PR TITLE
Add reference content, and move Lens docs to their own page

### PR DESCRIPTION
**Pull request by wylieconlon**
@KOTungseth here's the update PR that we talked about

### CODE REVIEW

File: docs/user/dashboard/dashboard.asciidoc [hunk 1]
```diff
@@ -98,444 +98,141 @@ To add a saved panel, click *Add from library*, then select the panel you want t
 TIP: To access your saved panels, open the main menu, then click *Visualize*.
 
 [float]
-[[create-panels-with-lens]]
-== Tutorial: Create a dashboard of panels with web server data
```
Comment (wylieconlon) [inline docs/user/dashboard/dashboard.asciidoc:102]: This was copy+pasted into a separate page. Seems a little easier to organize as an `include` if we want to move it back.

File: docs/user/dashboard/dashboard.asciidoc [hunk 2]
```diff
@@ -98,444 +98,141 @@ To add a saved panel, click *Add from library*, then select the panel you want t
...
 [role="screenshot"]
-image::images/lens_end_to_end_3_1_1.gif[Zoom in on the data]
+image::dashboard/images/dashboard-filters.png[Labeled interface with semi-structured search, time filter, and additional filters]
 
-. Change the *timestamp* interval.
+Semi-structured search::
+  Combines free text search with field-based search using the <<kuery-query,Kibana Query Language>>.
+  Type a search term to match across all fields, or begin typing a field name to
+  get prompted with field names and operators you can use to build a structured query.
+  +
```
Comment (KOTungseth) [inline docs/user/dashboard/dashboard.asciidoc:132]:
```suggestion

```

File: docs/user/dashboard/dashboard.asciidoc [hunk 3]
```diff
@@ -98,444 +98,141 @@ To add a saved panel, click *Add from library*, then select the panel you want t
...
-+
-[role="screenshot"]
-image::images/lens_end_to_end_3_1.png[Customize time interval]
+Time filter::
+  Dashboards have a global time filter which restricts the data shown, but individual panels can
+  override the time range.
+  The time range can be updated by dragging on a panel which shows time on the X axis.
 
-. From the *Chart Type* dropdown, select *Area*.
+  To override the time range for a specific panel:
```
Comment (KOTungseth) [inline docs/user/dashboard/dashboard.asciidoc:146]:
```suggestion
To override the time range for a specific panel:
```

File: docs/user/dashboard/dashboard.asciidoc [hunk 4]
```diff
@@ -98,444 +98,141 @@ To add a saved panel, click *Add from library*, then select the panel you want t
...
-display the *Unique visitors* data along a second y-axis, then change the formatting. When functions contain multiple formats, separate axes are created by default. 
-
-. From the editor, click *Unique visitors* in the line chart layer.
-
-. For *Axis side*, click *Right*, then click *Close*.
+image:images/time_range_per_panel.gif[Time range per dashboard panel]
+
+Additional filters with AND::
+  Individual filters can be added to a single dashboard or pinned to multiple pages in {kib}. Filters can be added using a simple editor or an advanced JSON editor for the {es} {ref}query-dsl.html[query DSL].
+  +
```
Comment (KOTungseth) [inline docs/user/dashboard/dashboard.asciidoc:157]:
```suggestion

```

File: docs/user/dashboard/dashboard.asciidoc [hunk 5]
```diff
@@ -98,444 +98,141 @@ To add a saved panel, click *Add from library*, then select the panel you want t
...
+  If more than one index pattern is shown on the dashboard, the filter editor will let you filter only one of them.
+  +
```
Comment (KOTungseth) [inline docs/user/dashboard/dashboard.asciidoc:159]:
```suggestion

```

File: docs/user/dashboard/dashboard.asciidoc [hunk 6]
```diff
@@ -98,444 +98,141 @@ To add a saved panel, click *Add from library*, then select the panel you want t
...
+  Filters can be dynamically added when you click a series in the dashboard.
+  +
```
Comment (KOTungseth) [inline docs/user/dashboard/dashboard.asciidoc:161]:
```suggestion

```

File: docs/user/dashboard/dashboard.asciidoc [hunk 7]
```diff
@@ -98,444 +98,141 @@ To add a saved panel, click *Add from library*, then select the panel you want t
...
+=== Download panel data as CSV
 
-.. From the editor, click *Top values of geo.src*.
+Most panels can be individually downloaded as CSV, but there is a shortcut available
+for *Lens* panels.
 
-.. From the *Advanced* dropdown, deselect *Group other values as "Other"*, then click *Close*.
+[role="xpack"]
+Lens::
+. Open the *Lens* panel menu, then select *More > Download as CSV*.
```
Comment (KOTungseth) [inline docs/user/dashboard/dashboard.asciidoc]:
```suggestion
Open the *Lens* panel menu, then select *More > Download as CSV*.
```

File: docs/user/dashboard/advanced-editors.asciidoc [hunk 8]
```diff
@@ -50,18 +51,15 @@ For detailed *Vega* and *Vega-Lite* information and examples, refer to <<resourc
...
-*Timelion* is a time series data visualizer that enables you to combine independent data sources within a single visualization.
-On *Timelion* visualizations, your data is displayed along a scale that changes color according to where your data falls on the specified scale. 
-Use the gauge to show how metric values relate to reference threshold values, or determine how a specified field is performing versus how it is expected to perform.
-
 *Timelion* is driven by a simple expression language that you use to:
 
-* Retrieve time series data
-* Perform calculations to find the answers to complex questions
+* Retrieve time series data from one or more indices
+* Perform math across time series
```
Comment (KOTungseth) [inline docs/user/dashboard/advanced-editors.asciidoc]:
```suggestion
* Perform math across two or more time series
```

File: docs/user/dashboard/aggregation-reference.asciidoc [hunk 9]
```diff
@@ -1,10 +1,104 @@
 [[aggregation-reference]]
-== Aggregation reference
+== Feature comparison
```
Comment (KOTungseth) [inline docs/user/dashboard/aggregation-reference.asciidoc]: Can we call this something along the lines of `Supported features by panel type`?

**Review by KOTungseth — APPROVED** [review]
These changes are great! I made a few minor suggestions, but otherwise, LGTM. 

The only part I'm having troubles with is the poor `Vega reference` section. It sticks out, but I'll work on a fix for that next.

**Pull request merged**